src/honeysql/core.cljc

```diff
@@ -3,15 +3,15 @@
   (:require [honeysql.format :as format]
             [honeysql.types :as types]
             [honeysql.helpers :refer [build-clause]]
-            [honeysql.util :refer [defalias]]
+            #?(:clj [honeysql.util :refer [defalias]])
```

Comment:
If we are using simply `def` in CLJS I'm wondering what the value is in having `defalias` in the first place...?